package.json

```diff
@@ -41,7 +41,7 @@
     "federalist": "npx gulp build",
     "lint": "npx gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files \"//select-a-language//,//2017//,//2018//,//2019//,//code-guidelines//\" ./_site",
+    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files '/2017/,/2018/,/code-guidelines/' ./_site",
```

Comment:
## Status code 200 and code-guidelines

Opted to maintain `code-guidelines` rather than ignoring code `200` to ensure we catch incorrect hashes on external links

Reply:
@mahoneycm shouldn't this be `/documentation/code-guidelines/`? The current url returns a 404.

Reply:
@mejiaj Since we're specifying file paths and not urls here, having `code-guidelines` is enough to properly target the correct page. Testing both and comparing the number of files ran as well as the test results shows no difference with either option.

Would you prefer the added specificity of `/documentation/code-guidelines/` ?

Reply:
Thanks for that note. I wasn't aware about file paths. Yes, I think so since it's the more accurate option.

FWIW on site build I still see this structure:

```bash
.
└── _site
    └── documentation
        └── code-guidelines
            └── index.html
```


![image](https://github.com/uswds/uswds-site/assets/3385219/e314a582-a684-4b29-9d08-e9807fb59987)

Reply:
Resolved in [9828b79](https://github.com/uswds/uswds-site/pull/2588/commits/9828b798f554523a3c80f243f4059e56d95f6704). 

I gave the `2017, 2018` links the same treatment for consistency 👍